build/AzurePipelinesTemplates/WindowsAppSDK-Build-Stage.yml

```diff
@@ -57,16 +57,42 @@ stages:
           buildConfiguration: 'release'
           normalizedConfiguration: 'fre'
     variables:
-      ob_outputDirectory: '$(REPOROOT)\out'
-      ob_sdl_codeSignValidation_excludes: '-|**\Release\**;-|**\packages\**'
-      ob_artifactBaseName: "FoundationBinaries_$(buildConfiguration)_$(buildPlatform)"
-      ${{ if parameters.runStaticAnalysis }}:
-        ob_sdl_apiscan_enabled: true
-      ${{ if not( parameters.runStaticAnalysis ) }}:
-        ob_sdl_apiscan_enabled: false
-      ob_sdl_apiscan_softwareFolder: '$(build.SourcesDirectory)\APIScanTarget'
-      ob_sdl_apiscan_symbolsFolder: '$(build.SourcesDirectory)\APIScanTarget;SRV*https://symweb.azurefd.net'
+      - name: ob_outputDirectory
+        value: '$(REPOROOT)\out'
+      - name: ob_sdl_codeSignValidation_excludes
+        value: '-|**\Release\**;-|**\packages\**'
+      - name: ob_artifactBaseName
+        value: "FoundationBinaries_$(buildConfiguration)_$(buildPlatform)"
+      - name: ob_sdl_apiscan_enabled
+        ${{ if parameters.runStaticAnalysis }}:
+          value: true
+        ${{ else }}:
+          value: false
+      - name: ob_sdl_apiscan_softwareFolder
+        value: '$(build.SourcesDirectory)\APIScanTarget'
+      - name: ob_sdl_apiscan_symbolsFolder
+        value: '$(build.SourcesDirectory)\APIScanTarget;SRV*https://symweb.azurefd.net'
+      - template: AzurePipelinesTemplates/WindowsAppSDK-Global-LKGVersions.yml@WindowsAppSDKConfigSourceBranch
```

Comment:
The new "- template: AzurePipelines..." line forced the remaining items in this section to be converted into the "- name: ..." format because mixing of the two formats in the same section is not permitted.